packages/integrations/partytown/README.md

```diff
@@ -61,37 +81,81 @@ If you open the "Network" tab from [your browser's dev tools](https://developer.
 
 ## Configuration
 
-### config.debug
+To configure this integration, pass a 'config' object to the `partytown()` function call in `astro.config.mjs`.
+
+__astro.config.mjs__
+```js
+...
+export default defineConfig({
+  integrations: [partytown({
+    config: {
+      //options go here
+    }
+  })]
+});
+```
+
+This mirrors the [Partytown config object](https://partytown.builder.io/configuration), but only `debug` and `forward` are exposed by this integration.
+
+<details>
+  <summary><strong>config.debug</strong></summary>
+  
+  <br/>
+
+  Partytown ships with a `debug` mode; enable or disable it by passing `true` or `false` to `config.debug`. If [`debug` mode](https://partytown.builder.io/debugging) is enabled, it will output detailed logs to the browser console. 
 
-You can set debug mode using this integration's `config.debug` option. If `config.debug` is unset, it will fall back to `true` if the command is `dev`.
+  If this option isn't set, `debug` mode will be on by default in [dev](https://docs.astro.build/en/reference/cli-reference/#astro-dev) or [preview](https://docs.astro.build/en/reference/cli-reference/#astro-preview) mode. 
+
+__astro.config.mjs__
 
 ```js
-// astro.config.mjs
-export default {
+export default defineConfig({
   integrations: [partytown({
     // Example: Disable debug mode.
     config: { debug: false },
-  })],
-}
+  
+})
 ```
+</details>
+
+<details>
+  <summary><strong>config.forward</strong></summary>
+  
+  <br/>
+  
+  Third-party scripts typically add variables to the `window` object so that you can communicate with them throughout your site. But when a script is loaded in a web-worker, it doesn't have access to that global `window` object.
 
-### config.forward
+  To solve this, Partytown can "patch" variables to the global window object and forward them to the appropriate script.
 
-Because we’re moving third-party scripts to a web worker, the main thread needs to know which variables to patch on window, and when these services are called, the data is correctly forwarded to the web worker. You can to set it on the `config.forward` option.
+  You can specify these variables on the `config.forward` option. [Read more in Partytown's documentation.](https://partytown.builder.io/forwarding-events)
```

Comment:
Be explicit instead of using “these”?

```suggestion
  You can specify which variables to forward with the `config.forward` option. [Read more in Partytown's documentation.](https://partytown.builder.io/forwarding-events)
```